### PR TITLE
chore: update changelog for version 2.10.7

### PR DESCRIPTION
**Pull request by erikayasuda**
- [x] update changelog for version 2.10.7

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
...
 Changelogs for versions not listed here can be found at https://github.com/DataDog/dd-trace-py/releases
 
+---
+
+## 2.10.7
+
+
+### Bug Fixes
+
+- CI Visibility: This fix resolves an issue where exceptions other than timeouts and connection errors raised while fetching the list of skippable tests for ITR were not being handled correctly and caused the tracer to crash.
```
Comment (erikayasuda) [inline CHANGELOG.md:12]:
```suggestion
- CI Visibility: Resolves an issue where exceptions other than timeouts and connection errors raised while fetching the list of skippable tests for ITR were not being handled correctly and caused the tracer to crash.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,19 @@
...
+- CI Visibility: Fixes a bug where `.git` was incorrectly being stripped from repository URLs when extracting service names, resulting in `g`, `i`, or `t` being removed (eg: `test-environment.git` incorrectly becoming `test-environmen`)
+- openai: Fixes a bug where \`asyncio.TimeoutError\`s were not being propagated correctly from canceled OpenAI API requests.
```
Comment (erikayasuda) [inline CHANGELOG.md:14]:
```suggestion
- openai: Fixes a bug where `asyncio.TimeoutError`s were not being propagated correctly from canceled OpenAI API requests.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,19 @@
...
+- profiling: Fixes endpoing profiling for stack v2, that is when `DD_PROFILING_STACK_V2_ENABLED` set.
```
Comment (erikayasuda) [inline CHANGELOG.md:15]:
```suggestion
- profiling: Fixes endpoing profiling for stack v2 when `DD_PROFILING_STACK_V2_ENABLED` set.
```

**Review by taegyunkim — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -4139,7 +4152,7 @@ Major changes to context management. See the upgrade section for the specifics.
 
 - Support for MySQL-python has been removed.
 
-- Support for psycopg \< 2.7 has been removed.
+- Support for psycopg \< 2.7 has been removed.
```
Comment (erikayasuda) [inline CHANGELOG.md:4155]:
```suggestion
- Support for psycopg2.7 has been removed.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -5968,7 +5981,7 @@ Read the [full changeset](https://github.com/DataDog/dd-trace-py/compare/v0.14.1
 ## 0.14.1 (25/09/2018)
 **Bug fixes**
 - [opentracer] Activate span context on extract (#606, #608)
-- [opentracer] Fix "does not provide the extra opentracing" (#611, #616)
+- [opentracer] Fix "does not provide the extra opentracing" (#611, #616)
```
Comment (erikayasuda) [inline CHANGELOG.md:5984]:
```suggestion
- [opentracer] Fix "does not provide the extra opentracing" (#611, #616)
```

**Comment by erikayasuda** [comment]
Restarting this PR because I messed it up

**Pull request closed without merging**